contracts/zaps/DepositZapMorphoAaveV2USD.vy

```diff
@@ -211,94 +210,70 @@ def remove_liquidity_imbalance(
     """
     fee: uint256 = CurveBase(BASE_POOL).fee() * BASE_N_COINS / (4 * (BASE_N_COINS - 1))
     fee += fee * FEE_IMPRECISION / FEE_DENOMINATOR  # Overcharge to account for imprecision
+    # TODO: what to do with the fee??
```

Comment:
it must be removed from the shares in my understanding of the code template

Reply:
Removed the fee according to a dev from Curve: pure zaps should not take fees